solana-programs/staking-bridge/programs/staking-bridge/src/lib.rs

```diff
@@ -35,7 +35,7 @@ use crate::wormhole::{
 #[cfg(not(feature = "no-entrypoint"))]
 use solana_security_txt::security_txt;
 
-declare_id!("HEDM7Zg7wNVSCWpV4TF7zp6rgj44C43CXnLtpY68V7bV");
+declare_id!("stkB5DZziVJT1C1VmzvDdRtdWxfs5nwcHViiaNBDK31");
```

Comment:
Same here (though I guess this isn't production anyway?)

Reply:
this is actually the prod one, so probably right. not sure what our convention here should be tbh

Reply:
maybe ill just commit all prod values

Reply:
Yeah I think that makes sense